config/locales/en/errors.yml

```diff
@@ -83,9 +83,18 @@ en:
           attributes:
             case_type:
               inclusion: Select a case type
+        steps/case/appeal_details_form:
+          attributes:
             appeal_with_changes_details:
-              blank: Add the details of what has changed
-              present: Details of what has changed are required for Appeal to Crown Court with changes to financial circumstances only
+              blank: Enter details of changes in financial circumstances
+            appeal_lodged_date:
+              blank: Enter the date the appeal was lodged
+              invalid: Enter a valid appeal lodged date
```

Comment:
Maybe just `Enter a valid date`?

Reply:
Yeah, I had my doubts, in some places we use just "enter a valid date", and in others like the hearing, we use "Enter a valid next hearing date".

In theory according to the design guidelines we should be explicit about what date it is, however given in this page (and other pages) there is only 1 date input (as opposite to the offences page for example) I agree "Enter a valid date" would be appropriate.
I can change it.